patches/coreclr/0001-glibc-sysctl-deprecation-handling.patch

```diff
@@ -0,0 +1,31 @@
+From f2a4abfda4f2dea0d1bece590222b304b6242cf3 Mon Sep 17 00:00:00 2001
+From: adaggarwal <aditya.aggarwal@microsoft.com>
+Date: Fri, 6 Mar 2020 18:25:45 +0000
+Subject: [PATCH] glibc sysctl deprecation handling
```

Comment:
I proposed this in coreclr and they suggested an alternative fix. That discussion is here: https://github.com/dotnet/coreclr/pull/28010.  This is the final fix merged into release/3.1: https://github.com/dotnet/coreclr/pull/28012

Reply:
Thanks for the pointers.

Reply:
We'll make sure to add the warning work-around patch if we don't get updated coreclr with the fix in next release.

Reply:
I am not sure if I got that fix into 2.1. Should I look into getting it in there too?

Reply:
Yep. That would help in getting rid of the patch next release.